### PR TITLE
support hybrid training service v2.0 config

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: nni/tools/nnictl/launcher.py [hunk 1]
```diff
@@ -592,16 +592,21 @@ def create_experiment(args):
...
-    except Exception:
-        pass
+    
     try:
         validate_all_content(experiment_config, config_path)
-    except Exception as e:
-        print_error(e)
-        exit(1)
+    except Exception:
+        print_warning('Validation with V1 schema failed. Trying to convert from V2 format...')
```
Comment (SparkSnail) [inline nni/tools/nnictl/launcher.py:599]: When use V2 yaml file, there always be a warning here?

Comment (QuanluZhang) [reply nni/tools/nnictl/launcher.py]: yes, this is a content that I merge from yuge's pr. I think it is okay for now

**Review by SparkSnail — APPROVED** [review]

**Review by liuzhe-lz — APPROVED** [review]

File: examples/trials/mnist-tfv1/config_hybrid.yml [hunk 2]
```diff
@@ -7,6 +7,8 @@ trainingServicePlatform: hybrid
 searchSpacePath: search_space.json
 #choice: true, false
 useAnnotation: false
+nniManagerIp: 10.150.144.59
+versionCheck: false
```
Comment (QuanluZhang) [inline examples/trials/mnist-tfv1/config_hybrid.yml:11]: remove ip and version check

File: nni/experiment/experiment.py [hunk 3]
```diff
@@ -171,6 +171,7 @@ def run(self, port: int = 8080, debug: bool = False) -> bool:
             while True:
                 time.sleep(10)
                 status = self.get_status()
+                print('zqlllll: ', status)
```
Comment (SparkSnail) [inline nni/experiment/experiment.py:174]: What's this line used for?

Comment (QuanluZhang) [reply nni/experiment/experiment.py:174]: sorry, forgot to remove it...

**Pull request merged**